docs/Tutorial.md

```diff
@@ -887,13 +914,13 @@ React-admin calls the Data Provider with one method for each of the actions of t
 The code for a Data Provider for the `my.api.url` API is as follows:
 
 ```js
-import { fetchUtils } from 'react-admin';
+import { fetchUtils } from "react-admin";
 import { stringify } from 'query-string';
 
 const apiUrl = 'https://my.api.com/';
 const httpClient = fetchUtils.fetchJson;
 
-const dataProvider= {
+export const dataProvider= {
```

Comment:
Should we change this to TS as well? Like declaring `dataProvider` as being of type `DataProvider`?

Reply:
No, all the snippets in the tutorial are in JS.